lib/FeedExpander.php

```diff
@@ -48,7 +48,9 @@ protected function collect_RSS_1_0_data($rssContent, $maxItems){
 		$this->load_RSS_2_0_feed_data($rssContent->channel[0]);
 		foreach($rssContent->item as $item){
 			debugMessage('parsing item ' . var_export($item, true));
-			$this->items[] = $this->parseItem($item);
+			if (null !== $this->parseItem($item)) {
```

Comment:
`$this->parseItem($item)` will return an empty array for native types (see `parseRSS_1_0_Item` for example), so instead of checking for null check if the array is empty: `if(empty($this->parseItem($item)){`